charts/questions.yaml

```diff
@@ -67,113 +69,113 @@ questions:
   show_subquestion_if: true
   subquestions:
   - variable: monitoring.version
-    default: 
+    default:
     label: "Monitoring Version"
     description: "Choose chart version of monitoring. If empty latest version will be installed"
     type: string
     group: "Monitoring"
   - variable: monitoring.values
-    default: 
+    default:
     label: "Monitoring Values"
     description: "Custom monitoring chart values"
     type: multiline
     group: "Monitoring"
-- variable: nodepool.name
-  default: 
+- variable: nodepools.0.name
```

Comment:
Is there a way to loop through the nodepools instead of only using the first one?

Reply:
When Daishan was working on this, the answer he gave me was no, its not possible to loop through for the questions.yaml . I was talking with @bashofmann about this solution and if my understanding is correct this is just a limitation of the tool at this time.